pulsar-client-cpp/lib/CMakeLists.txt

```diff
@@ -16,7 +16,8 @@
 
 file(GLOB PULSAR_SOURCES *.cc lz4/*.c checksum/*.cc)
 
-execute_process(COMMAND /bin/cat ../pom.xml COMMAND /bin/grep -Po "<?<version>[^<]+" COMMAND /bin/sed "s/.*>//g"  COMMAND /usr/bin/head -1 COMMAND /usr/bin/tr -d '\\\n' OUTPUT_VARIABLE PV)
+execute_process(COMMAND cat ../pom.xml COMMAND xmllint --format - COMMAND sed "s/xmlns=\".*\"//g" COMMAND xmllint --stream --pattern /project/version --debug - COMMAND grep -A 2 "matches pattern" COMMAND grep text COMMAND sed "s/.* [0-9] //g" OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE PV)
```

Comment:
@saandrews - yes 14.04 LTS man pages I found on the internet has xmllnt in them 

http://manpages.ubuntu.com/manpages/trusty/man1/xmllint.1.html